feature/aft/aft_base/otg_tests/aft_prefixcounters/README.md

```diff
@@ -0,0 +1,192 @@
+# AFT-2.1: AFTs Prefix Counters
+
+## Summary
+
+IPv4/IPv6 prefix counters
+
+## Procedure
+
+Configure DUT:port1 for IS-IS session with ATE:port1
+*   IS-IS must be level 2 only with wide metric.
+*   IS-IS must be point to point.
+*   Send 1000 ipv4 and 1000 ipv6 IS-IS prefixes from ATE:port1 to DUT:port1.
+*   IS-IS must be up and running, prefixes must be in RIB and FIB.
```

Comment:
Same question-- what is the intended means to verify this you were thinking?

Reply:
updated

Reply:
It still says the same as before? 
`*   IS-IS must be up and running, prefixes must be in RIB and FIB.` (which is not what we want to say here.)
But I do see you updated the other README. Can we update this one as well?

Reply:
removed